src/backend/controllers/tankController.js

```diff
@@ -73,6 +73,36 @@ exports.favoriteTank = async (req: Request, res: Response) => {
 	});
 }
 
+exports.unfavoriteTank = async (req: Request, res: Response) => {
+	const errors = validationResult(req);
+	
+	if (!errors.isEmpty()) {
+		// 400 is a bad request
+		console.error('Could not handle request');
+
+		return res
+			.status(400)
+			.json({ errors: errors.array() });
+	}
+
+	// Find the user and set favoriteTank to null.
```

Comment:
If they currently have a wager going on, you should give them the money for that wager back